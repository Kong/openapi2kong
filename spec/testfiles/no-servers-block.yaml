openapi: "3.0.0"
info:
  title: Simple API overview
  version: v2
# Test file without a servers block
#servers:
#  - url: http://backend.com/path
paths:
  /v2:
    get:
      operationId: getVersionDetailsv2
      summary: Show API version details
      responses:
        '200':
          description: |-
            200 response
        '203':
          description: |-
            203 response
