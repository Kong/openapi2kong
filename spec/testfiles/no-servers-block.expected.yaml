---
services:
- host: Simple_API_overview
  path: /
  tags:
  - OAS3_import
  - OAS3file_no-servers-block.yaml
  name: Simple_API_overview
  routes:
  - methods:
    - GET
    strip_path: false
    tags:
    - OAS3_import
    - OAS3file_no-servers-block.yaml
    name: Simple_API_overview-path-get
    paths:
    - /v2$
upstreams:
- tags:
  - OAS3_import
  - OAS3file_no-servers-block.yaml
  name: Simple_API_overview
  targets: {}
_format_version: '1.1'
...
